src/containers/layout/navbar/NavBar.tsx

```diff
@@ -99,10 +97,16 @@ const Navbar = () => {
     )
   })
 
+  const scrollToWelcome = (): void => {
+    const welcome = document.getElementById(guestRoutes.welcome.route)
+    if (welcome) welcome.scrollIntoView({ behavior: 'smooth' })
+  }
+
```

Comment:
In React we try do not use `getElementById`. Look at `findOffersMenu` (59th row) items have scroll behavior.